functions/Remove-DbaPlanCache.ps1

```diff
@@ -0,0 +1,89 @@
+﻿function Remove-DbaPlanCache {
+    <#
+        .SYNOPSIS
+            Removes adhoc and prepared plan caches is single use plans are over defined 
+            threshold
+    
+        .DESCRIPTION
+            "Checks ahoc and prepared plan cache for each database, if over 100 MBs 
+            removes from the cache."
+    
+            This command automates that process.
+    
+            References: https://www.sqlskills.com/blogs/kimberly/plan-cache-adhoc-workloads-and-clearing-the-single-use-plan-cache-bloat/
+    
+            Note: This command removes the plans from all SQL instances on the destionation 
+            server but the process column is specific to -SqlInstance passed.
+    
+        .PARAMETER SqlInstance
+            Allows you to specify a comma separated list of servers to remove the plans.
+    
+        .PARAMETER SqlCredential
+            Allows you to login to the SQL instance using alternative credentials.
+    
+        .PARAMETER Threshold
+            Memoery used threshold.
+    
+        .PARAMETER WhatIf
+            If this switch is enabled, no actions are performed but informational messages will be displayed that explain what would happen if the command were to run.
```

Comment:
Need to also add the help information for `Confirm` since we are supporting ShouldProcess. This is one of the failures in the test validations in Appveyor:

```
Name     : It gets help for parameter: Confirm : in Remove-DbaPlanCache
Result   : Failed
Message  : Expected a value, but got $null or empty.
```

From the template can simply add this:
```powershell
        .PARAMETER Confirm
            If this switch is enabled, you will be prompted for confirmation before executing any operations that change state.
```